lib/tds.js

```diff
@@ -348,9 +381,13 @@ function finalizeDNRRulesAndLookup (startingRuleId, dnrRules) {
         // just add the rule to the ruleset and match details to the lookup now.
         if (!rule.condition.requestDomains ||
             rule.priority !== BASELINE_PRIORITY ||
-            rule.action === 'redirect') {
+            rule.action === 'redirect' || rule[inverseCustomActionSymbol]) {
```

Comment:
Nit: Would you mind moving this onto a new line and also adding a note about this case in the above comment? Just so that it's clear the inverse rule is something used by Click to Load and not related to surrogate script redirection.

Reply:
👍